pyuvdata/uvdata/uvdata.py

```diff
@@ -3473,7 +3478,7 @@ def __iadd__(
 
     def fast_concat(
         self,
-        other,
+        others,
```

Comment:
I am slightly concerned about changing the name of a positional argument. While not necessary it _can_ be called as a keyword argument in python. I think it would be wise to leave the `other` keyword in the function call, but then throw a deprecation warning stating `others` is the prefered call signature and `other` will stop working (in some release). The `other` parameter can then just be assigned to `others` inside.

Reply:
Good point. Should I add `other` as a named parameter then in order to catch someone using it with it's name?

Reply:
I think that sounds pretty easy. Just cast the default to None and check if it gets populated during a call?

Reply:
So it turns out this doesn't quite work. I added `other` as a named parameter, but if you pass it but don't pass anything for `others` it errors with `TypeError: fast_concat() missing 1 required positional argument: 'others'`.

Reply:
We can just leave it named `other` even though it takes a list. That has the advantage of not changing the API but it makes the parameter naming less clear. Is there a way in python to tell if something was passed by position or name?

Reply:
That's a good point. I had not considered TypeError from missing a positional. We definitely could leave it, or change it. Probably won't make much difference in the end. We could always make a lot of noise when the release comes out to warn about the api change. I'm not sure if there is a way to tell honestly.

Reply:
Hmm, I did some googling and can't tell that there's a way to detect calling a positional argument by name. Interestingly, starting in python 3.8 you can define parameters as positional only (you can already define parameters as keyword only in python 3.*).

Reply:
I think it's better to keep the API as it is, and just point out in the docstring that we support either a single object or an iterable for `other` (as we do now). But I don't feel super strongly, we could also change the API and make lots of noise during the release process.

Reply:
I just put it back to `other` to avoid the API change.